src/Symfony/Bridge/Doctrine/Tests/ArgumentResolver/EntityValueResolverTest.php

```diff
@@ -0,0 +1,666 @@
+<?php
+
+namespace Symfony\Bridge\Doctrine\Tests\ArgumentResolver;
+
+use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\Persistence\ManagerRegistry;
+use Doctrine\Persistence\Mapping\ClassMetadata;
+use Doctrine\Persistence\Mapping\ClassMetadataFactory;
+use Doctrine\Persistence\ObjectManager;
+use Doctrine\Persistence\ObjectRepository;
+use LogicException;
```

Comment:
CS: We usually don't import from the global namespace.

Reply:
good catch! 

this was copy/pasted from Doctrine repo